data/settings/actions.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  REMOVE_TOKEN_FROM_BLACKLIST,
+  ADD_TOKEN_TO_BLACKLIST
+} from "./constants";
+
+const addTokenToBlackList = (tokenID: string) => ({
```

Comment:
Minor:  Change all instances of `tokenID` to `tokenId` to match conventions throughout the rest of the project